content/automate/ManageIQ/Infrastructure/VM/Transform/Import.class/__methods__/list_clusters.rb

```diff
@@ -0,0 +1,45 @@
+module ManageIQ
+  module Automate
+    module Infrastructure
+      module VM
+        module Transform
+          module Import
+            class ListClusters
+              def initialize(handle = $evm)
+                @handle = handle
+              end
+
+              def main
+                values_hash = {}
+                values_hash[nil] = '-- select cluster from list --'
+
+                provider_id = @handle.root['dialog_provider']
+                if !provider_id.nil? && !provider_id.empty? && provider_id != '!'
```

Comment:
@matobet you could use if provider.present? instead of !provider_id.nil? && !provider_id.empty?